src/frontend/packages/lib_common/src/queries/actionOne.tsx

```diff
@@ -29,5 +39,5 @@ export const actionOne: MutationFunction<
     throw { code: 'exception' };
   }
 
-  return await response.json();
+  return (await response.json()) as Promise<T>;
```

Comment:
Why is it needed to cast here while in the function signature, the return type is already `Promise<T>` ?

Reply:
As talk, `response.json()` return `Promise<any>`, that give an error.